providers/fips/fipsprov.c

```diff
@@ -804,6 +804,9 @@ static const OSSL_ALGORITHM fips_keyexch[] = {
 static const OSSL_ALGORITHM fips_signature[] = {
 #ifndef OPENSSL_NO_DSA
     { "DSA:dsaEncryption", "provider=fips,fips=yes", dsa_signature_functions },
+#endif
+#ifndef OPENSSL_NO_EC
+    { "ECDSA:id-ecPublicKey", "provider=fips,fips=yes", ecdsa_signature_functions },
```

Comment:
Please remove `:id-ecPublicKey` (it should be removed from the "ECDH" line as well).

Reply:
To compensate, please fix these lines:

https://github.com/openssl/openssl/blob/ca7f7b951825e23dddb798f6a61f50a04225d25a/providers/implementations/keymgmt/ec_kmgmt.c#L52-L55

Reply:
Done

Reply:
Muuuch better, thank you